src/main/java/org/mitre/synthea/export/TextExporter.java

```diff
@@ -263,18 +532,44 @@ private static void encounter(List<String> textRecord, Encounter encounter) {
    * @param condition
    *          The condition to add to the export
    */
-  private static void condition(List<String> textRecord, Entry condition) {
+  private static void condition(List<String> textRecord, Entry condition, Boolean end) {
     String start = dateFromTimestamp(condition.start);
     String stop;
-    if (condition.stop == 0L) {
-      //     "YYYY-MM-DD"
-      stop = "          ";
-    } else {
-      stop = dateFromTimestamp(condition.stop);
-    }
     String description = condition.codes.get(0).display;
+    if (end) {
+      if (condition.stop == 0L) {
+        //     "YYYY-MM-DD"
+        stop = "          ";
+      } else {
+        stop = dateFromTimestamp(condition.stop);
+      }
+      textRecord.add("  " + start + " - " + stop + " : " + description);
+    }
+    else {
+      textRecord.add("  " + start + " : " + description);
+    }
+  }
 
-    textRecord.add(start + " - " + stop + " : " + description);
+  /**
+   * Write a line for a condition that has not ended at the time of the encounter.
+   *
+   * @param textRecord
+   *          Text format record, as a list of lines
+   * @param condition
+   *          The condition to add to the export
+   * @param encounter
+   *          The encounter at which the continuing condition is reported
+   */
+  private static void conditionpast(List<String> textRecord, Entry condition, Encounter encounter) {
+    String start = dateFromTimestamp(condition.start);
+    if ((condition.stop == 0L || condition.stop > encounter.stop) && (condition.start < encounter.start)) {
+      //checks that the condition hasn't ended by the time of the encounter
+      //and began prior to the encounter
+      String description = condition.codes.get(0).display;
+        textRecord.add("  " + start + " : " + description);
+    } else {
+      return;
```

Comment:
Don't need to explicitly return in a case like this - you can just omit the `else`